docs/install/create-an-offline-installation-of-visual-studio.md

```diff
@@ -103,14 +103,13 @@ Start by downloading the Visual Studio 2019 bootstrapper from either the [Visual
 
 ::: moniker range=">=vs-2022"
 
->[!TIP]
-> Released versions of Visual Studio 2022 are not yet available, the bootstrappers below are for the preview release of Visual Studio 2022.
->Start by downloading the Visual Studio 2022 bootstrapper from the [Visual Studio downloads page](https://aka.ms/vs2022preview).
+
```

Comment:
Let's keep the following text: 
Start by downloading the Visual Studio 2022 bootstrapper from the [Visual Studio downloads page](https://visualstudio.microsoft.com/downloads).